bundles/org.openhab.voice.googlestt/pom.xml

```diff
@@ -14,146 +14,155 @@
   </properties>
   <dependencies>
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-speech</artifactId>
-      <version>2.2.2</version>
+      <version>4.7.0</version>
       <scope>compile</scope>
     </dependency>
     <!--cloud-speech deps -->
     <dependency>
       <groupId>com.google.api.grpc</groupId>
       <artifactId>proto-google-common-protos</artifactId>
-      <version>2.7.1</version>
+      <version>2.14.2</version>
       <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>com.google.http-client</groupId>
       <artifactId>google-http-client</artifactId>
-      <version>1.40.1</version>
+      <version>1.43.0</version>
       <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>com.google.auth</groupId>
       <artifactId>google-auth-library-credentials</artifactId>
-      <version>1.2.1</version>
+      <version>1.16.0</version>
       <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>com.google.auth</groupId>
       <artifactId>google-auth-library-oauth2-http</artifactId>
-      <version>1.3.0</version>
+      <version>1.4.0</version>
       <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>com.google.api.grpc</groupId>
       <artifactId>proto-google-cloud-speech-v1</artifactId>
-      <version>2.2.2</version>
+      <version>4.7.0</version>
       <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
-      <version>3.19.6</version>
+      <version>3.21.12</version>
       <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>com.google.api</groupId>
       <artifactId>gax</artifactId>
-      <version>2.8.1</version>
-      <scope>compile</scope>
+      <version>2.23.3</version>
     </dependency>
     <dependency>
       <groupId>com.google.api</groupId>
       <artifactId>gax-grpc</artifactId>
-      <version>2.8.1</version>
+      <version>2.23.3</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.api</groupId>
+      <artifactId>gax-httpjson</artifactId>
+      <version>0.108.3</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.10.1</version>
+    </dependency>
```

Comment:
Do you need this for the dependencies? Just checking, since it will add more than 200 kB to the bundle until this is upgraded in core, which is unfortunately not easy: openhab/openhab-core#3321.

Reply:
Seems to be required in that range. 
```
filter:="(&(osgi.wiring.package=com.google.gson)(version>=2.10.0)(!(version>=3.0.0)))"]]
```
I think I already tried to bundle without it and it failed but I'm not totally sure it was not the mentioned error and I mixed things.

Let me do a check.

Reply:
Seems to work ok without it!